homeassistant/components/braviatv/__init__.py

```diff
@@ -1,26 +1,31 @@
 """The Bravia TV component."""
+from __future__ import annotations
+
 import asyncio
 from datetime import timedelta
 import logging
+from typing import Final, Iterable
 
 from bravia_tv import BraviaRC
 from bravia_tv.braviarc import NoIPControl
 
 from homeassistant.components.media_player import DOMAIN as MEDIA_PLAYER_DOMAIN
 from homeassistant.components.remote import DOMAIN as REMOTE_DOMAIN
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_HOST, CONF_MAC, CONF_PIN
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers.debounce import Debouncer
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
 from .const import CLIENTID_PREFIX, CONF_IGNORED_SOURCES, DOMAIN, NICKNAME
 
 _LOGGER = logging.getLogger(__name__)
 
-PLATFORMS = [MEDIA_PLAYER_DOMAIN, REMOTE_DOMAIN]
-SCAN_INTERVAL = timedelta(seconds=10)
+PLATFORMS: Final = [MEDIA_PLAYER_DOMAIN, REMOTE_DOMAIN]
```

Comment:
Mypy can't automatically infer inner type of collections. Here it'll only get `Final[list[Any]]`
```suggestion
PLATFORMS: Final[list[str]] = [MEDIA_PLAYER_DOMAIN, REMOTE_DOMAIN]
```